docroot/modules/humsci/hs_migrate/hs_migrate.install

```diff
@@ -73,3 +73,76 @@ function hs_migrate_update_8006() {
     }
   }
 }
+
+/**
+ * Remove unused importers.
+ */
+function hs_migrate_update_8007() {
+  $unused_importers = [
+    'hs_d7_gallery_nodes',
+    'hs_d7_gallery_paragraphs',
+    'hs_d7_people',
+    'hs_d7_publications',
+    'hs_news_rss',
+    'custm_cesta_publications',
+    'custm_dlcl_research',
+    'custm_dlcl_research_unit_group',
+    'custm_em1060_manuscript',
+    'custm_facultyaffairs_users',
+    'custm_hs_d7_gc_publication_component',
+    'custm_hshr_client',
+    'custm_sgs_d7_degrees_offered',
+    'custm_sgs_d7_fund_opp',
+    'custm_sgs_d7_testimonials',
+    'custm_stats_tech_reports',
+    'custm_hs_wmlcp_users',
+    'hs_d7_news'
+  ];
+
+  // Get site name to prevent removing hs_d7_news from MRC.
+  $site_name = \Drupal::config('system.site')->get('name');
+  $is_mrc = str_contains($site_name, 'Mathematics Research Center');
```

Comment:
I'd prefer to find a more reliable way to determine the MRC site. Using the site directory or something like that is preferred.